tests/e2e/specs/shopper/shopper-wc-blocks-checkout-failures.spec.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+const { shopper, merchant, uiUnblocked } = require( '@woocommerce/e2e-utils' );
+
+/**
+ * Internal dependencies
+ */
+import {
+	shopperWCP,
+	merchantWCP,
+	describeif,
+	RUN_WC_BLOCKS_TESTS,
+} from '../../utils';
+
+const shippingDetails = config.get( 'addresses.customer.shipping' );
+const productName = config.get( 'products.simple.name' );
+
+import {
+	fillCardDetailsWCB,
+	confirmCardAuthentication,
+} from '../../utils/payments';
+
+describeif( RUN_WC_BLOCKS_TESTS )(
+	'WooCommerce Blocks > Checkout failures',
+	() => {
+		beforeAll( async () => {
+			await merchant.login();
+			await merchantWCP.addNewPageCheckoutWCB();
+			await merchant.logout();
+			await shopper.login();
+			await shopper.goToShop();
+			await shopper.addToCartFromShopPage( productName );
+			await shopperWCP.openCheckoutWCB();
+			await shopperWCP.fillShippingDetailsWCB( shippingDetails );
+		} );
+
+		afterEach( async () => {
+			// Reload the page after every test so there are no messages
+			await page.reload();
+			await uiUnblocked();
+			await page.waitFor( 2000 );
```

Comment:
It'd be nice to not wait for 2 seconds and instead wait for an specific change in the UI. Not a blocker though.

Reply:
Yes I tried with many elements but ended in flakiness again, so I used this simple wait which was the best solution to avoid flakiness. We could do some separate PR to exclude all waits and to use something else like a specific element, I agree to never use waits.